### PR TITLE
Add programs

### PR DESCRIPTION
**Pull request by Joseph-Developer**

### CODE REVIEW

File: configuration/programs/programs-ces.csv [hunk 1]
```diff
@@ -1,2 +1,9 @@
 Uuid,Name,Description,Void/Retire,Program concept,Outcomes concept
-${program.hearts.uuid},HEARTS,Programa HEARTS,,PIH:HEARTS Program,PIH:Generic outcome
```
Comment (mogoodrich) [inline configuration/programs/programs-ces.csv:2]: You should rename this "programs.csv" to have it override the "programs.csv" in config-pihemr, without having to remove it from config-pihemr.

File: constants.yml [hunk 2]
```diff
@@ -18,3 +18,17 @@ encounterType:
 program:
   hearts:
     uuid: "6cceab45-756f-427b-b2da-0e469d4a87e0"
```
Comment (mogoodrich) [inline constants.yml:20]: As long as you don't need to change the uuids for any reason (I don't think you do?) you don't need these changes, the build will pick up the uuids from the config-pihemr.

**Review by mogoodrich — CHANGES_REQUESTED** [review]

File: constants.yml (hunk 2 above)
Comment (brandones) [reply constants.yml:20]: I think it is preferable to not have the layer of mysterious indirection. Nothing is gained by having config-ces rely on config-pihemr for these UUIDs.

File: configuration/programs/programs-ces.csv (hunk 1 above)
Comment (brandones) [reply configuration/programs/programs-ces.csv:2]: @Joseph-Developer Debes cambiar el nombre de "programs-ces.csv" a "programs.csv".

**Review by brandones — APPROVED** [review]

**Pull request merged**